Cubical/HITs/S1/Base.agda

```diff
@@ -248,6 +260,62 @@ basedΩS¹≡Int : (x : S¹) → basedΩS¹ x ≡ Int
 basedΩS¹≡Int x = (basedΩS¹≡ΩS¹ x) ∙ ΩS¹≡Int
 
 
+---- Alternative proof of the same thing -----
```

Comment:
Which proof is better? If one is not as good then it should be put in a private

Reply:
I think my proof is the only proof that is being used anywhere else, so I guess I'll put the other one in private.